addon/webextension/background/loadSelector.js

```diff
@@ -25,19 +25,19 @@ window.loadSelector = (function () {
     let lastPromise = Promise.resolve(null);
     scripts.forEach((script) => {
       lastPromise = lastPromise.then(() => {
-        return chrome.tabs.executeScript({
+        return browser.tabs.executeScript({
           file: script
         }, () => {
-          if (chrome.runtime.lastError
-              && chrome.runtime.lastError.message != "Script returned non-structured-clonable data") {
-              catcher.unhandled(chrome.runtime.lastError, {script: script});
-              console.error("Error loading script", script, ":", chrome.runtime.lastError);
+          if (browser.runtime.lastError
+              && browser.runtime.lastError.message != "Script returned non-structured-clonable data") {
+              catcher.unhandled(browser.runtime.lastError, {script: script});
+              console.error("Error loading script", script, ":", browser.runtime.lastError);
```

Comment:
Hmm, actually it looks like [Promise.all should preserve array order](http://stackoverflow.com/questions/28066429/promise-all-order-of-resolved-values), so that may be a cleaner approach here.